vllm_ascend/worker/mtp_proposer_v1.py

```diff
@@ -166,12 +167,26 @@ def propose(
         else:
             num_input_tokens = num_tokens
 
-        attn_metadata = self.runner.attn_metadata_builder.build(
+        common_attn_metadata = AscendCommonAttentionMetadata(
+            query_start_loc=self.runner.query_start_loc[:batch_size + 1],
+            query_start_loc_cpu=self.runner.query_start_loc_cpu[:batch_size + 1],
```

Comment:
Should not use `query_start_loc` from `self.runner`, since mtp's `query_start_loc` could be different from the main model.
Instead, the correct way is  `query_start_loc=cu_num_tokens`